docs/src/manifolds/embedded.md

```diff
@@ -0,0 +1,43 @@
+# Embedded manifold
+
+Some manifolds can easily be defined by using a certain embedding. For example the [`Sphere`](@ref)`(n)` is embedded in [`Euclidean`](@ref)`(n+1)`.
+Similar to the metric and [`MetricManifold`](@ref), an embedding is often implicitly assumed.
+We introduce the embedded manifolds hence as an [`AbstractDecoratorManifold`](@ref).
+
+This decorator enables to use such an embedding in an transparent way. Different types of
+embeddings can be distinguished using the [`AbstractEmbeddingType`](@ref).
+
+The embedding also covers representation of tangent vectors.
+For both points and tangent vectors the function [`embed`](@ref) returns their representation in the embedding.
+For any point or vector in the embedding the functions [`project_point`](@ref) and [`project_tangent`](@ref) can be used to obtain the closest point on the manifold and tangent vector in the tangent space, respectively.
+A specific example where [`embed`](@ref) might be useful, is for example a Lie group, where tangent vectors are often represented in the Lie algebra.
+Then their representation is different from the representation in the embedding.
+
+## Isometric Embeddings
+
+For isometric embeddings the type [`AbstractIsometricEmbeddingType`](@ref) can be used to
+avoid reimplementing the metric. See [`Sphere`](@ref) or [`Hyperbolic`](@ref) for example.
+Here, the exponential map, the logarithmic map, the retraction and its inverse
```

Comment:
Isometric embedding is exactly that the metric restricted to the (embedded) tangent space is the one from the embedding, yes. In some cases this restriction also directly applies for geodesics (see symmetric or skew symmetric, this is the case `DefaultIsometricEmbedding`) but that's not necessarily the case (see sphere and hyperbolic).

Reply:
Primarily I mean that exponential maps etc. are `:intrasparent` for all embeddings, not only isometric ones as this paragraph implicitly suggests. Maybe this piece of information could be moved to general discussion of embeddings.

Reply:
Ah. Yes they are, because you basically have a constraint PDE to solve (instead of the unconstraint in the embedding). It's just this one special type where they are transparent. Yes I can add that to the general part.